server/src/config.ts

```diff
@@ -31,11 +31,17 @@ const conf = convict({
     default: 'debug',
     env: 'LOG_LEVEL',
   },
-  timeUpdate: {
-    doc: 'data refresh interval',
-    format: Number,
-    default: 1800000,
-    env: 'DATA_REFRESH_INTERVAL',
+  cronTimeExpression: {
+    doc: 'cron interval to refresh data',
+    format: String,
+    default: '59 23 * * *',
```

Comment:
interval was 1800000 milliseconds = 1800 seconds = 30 minutes
so, 'each 30 minutes' cron expression should be used:

`*/30 * * * *`